app/helpers/Intersect.js

```diff
@@ -100,6 +101,54 @@ export async function IntersectSet(concernLocationArray, completion) {
   });
 }
 
+// Function to determine if two location points are "nearby".
+// Uses shortcuts when possible, then the exact calculation.
+// returns boolean
+export function areLocationsNearby(lat1, lon1, lat2, lon2) {
```

Comment:
A couple of simple tests for general and edge cases would be super helpful.

Reply:
Added testing on the renamed function